src/components/astra-editor/index.ts

```diff
@@ -136,7 +136,7 @@ export class AstraEditor extends LitElement {
           syntaxHighlighting(defaultHighlightStyle, { fallback: true }),
           autocompletion(),
           highlightActiveLine(),
-          indentUnit.of('  '),
+          indentUnit.of('    '),
```

Comment:
🤐

Reply:
Didn't know we are 4 spaces fan :)

Reply:
I’m a 4 spacer haha, a pattern too hard for me to break 🫣